packages/scene-composer/src/components/three-fiber/AnimationComponent/AnimationComponent.tsx

```diff
@@ -0,0 +1,81 @@
+// eslint-disable-next-line import/default
+import React, { Fragment, useEffect } from 'react';
+import * as THREE from 'three';
+import { useFrame } from '@react-three/fiber';
+
+import { useSceneComposerId } from '../../../common/sceneComposerIdContext';
+import { ISceneNodeInternal, useEditorState, useViewOptionState, IAnimationComponentInternal } from '../../../store';
+import { IAnimationComponent } from '../../../interfaces';
+
+const toggleIsAnimationPaused = (
+  object: THREE.Object3D,
```

Comment:
Could you use https://threejs.org/docs/index.html#api/en/animation/AnimationMixer.getRoot to skip passing in the object and just pass in the AnimationMixer?

Reply:
yes